src/website/src/pages/ec/content-types/events/detail/docs/usage.md

```diff
@@ -3,4 +3,12 @@ title: Usage
 order: 1
 ---
 
-Templates allow you to quickly create web pages and websites that are in line with the European Commission's corporate guidelines.
+import { Paragraph, Anatomy, Link } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  The detail page provides comprehensive content on the <strong>practical information for the event</strong>. 
+</Paragraph>
```

Comment:
Well, it's an md file, but there is a space at the end of this line, think it's not intentional..